src/components/shared/TypeAhead.vue

```diff
@@ -111,19 +116,29 @@ export default {
             self.$emit('selection',result)
             self.query = '';
         },
+        scroll(arrowCounter) {
```

Comment:
I meant to mention this on zoom...instead of passing in the value, we could probably change it to just use the value already in vue (this.arrowCounter).  Not super critical though